config/develop/veoibd.yaml

```diff
@@ -1,14 +1,10 @@
 ---
 template_path: veoibd.yaml
-stack_name: veoibd
+stack_name: veoibd-develop
```

Comment:
You probably don't want to do, it's better to keep the same stack names in both AWS sandbox and AWS scicomp accounts for consistency. 

Also when  you change this it deploys another stack `veoibd-develop` to sandbox but also keeps the old one `veoibd` as well.  you should login to sandbox and manually remove the one you don't want.